utils/SnapPy/Snappy/BrowserWidget.py

```diff
@@ -24,12 +24,20 @@
     *******************************************************************
 """
 
-from PyQt5 import QtCore, QtWidgets, QtWebKitWidgets
+from PyQt5 import QtCore, QtWidgets
+try:
+    #Qt5.5 and earlier
+    from PyQt5.QtWebKitWidgets import QWebPage, QWebView
+except:
+    #Qt5.6 and later - QtWebKitWidgets is deprecated
+    from PyQt5.QtWebEngineWidgets import QWebEnginePage as QWebPage
+    from PyQt5.QtWebEngineWidgets import QWebEngineView as QWebView
```

Comment:
Wouldn't it be preferrable to flip these? Are early versions of QtWebEngineWidgets buggy? Could you catch only `ImportError` instead of catch all?